examples/medplum-chart-demo/src/bots/core/bot-utils.ts

```diff
@@ -0,0 +1,271 @@
+import { generateId, getReferenceString } from '@medplum/core';
+import {
+  BundleEntry,
+  BundleEntryRequest,
+  ClinicalImpression,
+  Coding,
+  Condition,
+  Encounter,
+  Observation,
+  ObservationComponent,
+  Patient,
+  Practitioner,
+  Quantity,
+  Reference,
+} from '@medplum/fhirtypes';
+
+interface ObservationData {
+  bloodPressure: {
+    systolic?: number;
+    diastolic?: number;
+  };
+  height?: Quantity;
+  weight?: Quantity;
+  bmi?: Quantity;
+  date?: string;
+}
+
+export interface GeneralObservationData extends ObservationData {
```

Comment:
each of these is specific to their own bot, so should live in the corresponding bot files